src/main/java/com/emc/pravega/perf/PerfStats.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License. You may obtain a copy of the License at
- * 
+ * <p>
```

Comment:
Please, remove this: I know that is how IntelliJ automatically formats the header, but this is not how the header is in all other files in Pravega.